dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -47,6 +47,9 @@ using namespace Microsoft::Windows::AppNotifications::Helpers;
 
 namespace winrt::Microsoft::Windows::AppNotifications::implementation
 {
+    // Must be static for AppNotificationManager::Deserialize
+    static wil::unique_event s_waitHandleForArgs;
```

Comment:
We need to move the Manager classes to the COM static store to ensure that we have no issues with cleanup if COM rundown occurs. Please file a separate bug for that.

Reply:
https://github.com/microsoft/WindowsAppSDK/issues/2183